src/StorageWriter/StorageWriter.ts

```diff
@@ -42,21 +56,31 @@ export class StorageWriter {
     this.router = this.container.get('Router')
     await this.router.start()
 
-    await this.createIndices()
+    this.service = this.container.get('Service')
+    await this.service.start()
+
+    this.daoClaims = this.container.get('DAOClaims')
+    await this.daoClaims.start()
 
     this.logger.info('StorageWriter Started')
   }
 
   async stop() {
+    this.logger.info('Stopping StorageWriter Service')
```

Comment:
what do you say if we add an empty line per action?

Reply:
Sounds good, adjusted, I think I understood correctly.